### PR TITLE
FS-597 zap update - FOR REVIEW - DO NOT MERGE YET

### PR DESCRIPTION
**Pull request by harryyo**

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -1,5 +1,5 @@
-ARG BASE_IMAGE_TAG="3.19.0-rc.835"
-FROM ghcr.io/xgovformbuilder/digital-form-builder-runner:$BASE_IMAGE_TAG as base
```
Comment (gidsg) [inline Dockerfile:2]: Could this be in the `communitiesuk` org rather than your personal one?

Comment (endamccormack) [reply Dockerfile:2]: Yeah I agree

Comment (endamccormack) [reply Dockerfile:2]: Should we not just revert the changes to this file?

**Comment by endamccormack** [comment]
This needs a description @harryyo. It seems wrong to me reading the PR but I dont know why you've done anything.

**Pull request closed without merging**